src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -137,14 +137,15 @@ class PanelMediaBrowser extends LitElement {
           --mdc-theme-primary: var(--app-header-text-color);
         }
         ha-media-player-browse {
-          height: calc(100vh - 64px);
+          height: calc(100vh - 48px);
```

Comment:
All these calculation changes seem weird, since the compact header is configurable. How will this work?

Reply:
For anything that isn't in Lovelace, the calculations needed changed to fit the new size of app-header. For anything in Lovelace I still need to do an adjustment.  Both `hui-masonry-view` and `hui-panel-view` need to change the calculation depending on compact mode.

Reply:
Since this is now a UI overhaul, there is no need to adjust masonry view or panel view. I will look at other areas that are using hui-root/app-header to see where adjustments might need made.

Reply:
So it looks like the only UIs that would be affected would be the ones that actually use tabs: dev-tools panel, mailbox panel, and Lovelace. They all seem to already handle the change.

Reply:
I wonder if we can make it all easier and just define the height of the toolbar in a var?

Reply:
Yeah, maybe like:```
--view-height-no-tabs: calc(100vh - 48px);
--view-height-tabs: calc(100vh - 96px);
```

Reply:
Thinking about this more, the header itself can only really be 2 heights 48px or 96px depending if tabs are used or not, but the calculation that views below the header would use could be completely different then what I have above.

While the current files changed in this PR just use the single header value of 48px, that could change in the future. So rather than storing a calculation as the vars, it seems to make more sense to store just the single pixel values (48px, 96px) and at that point  `calc(100vh - 48px)` seems cleaner than something like `calc(100vh - var(--header-no-tabs))` though slightly less "readable".

Should we go ahead and add something like below or is that overkill? ```
--header-no-tabs: 48px
--header-with-tabs: 96px
```

Reply:
Going to think on this and probably work it into a different PR as this one is just modifying already existing calculations. I think adding more vars than just ones that use `app-header`'s height would be useful.